MissionTable.cpp

```diff
@@ -28,30 +28,72 @@ void WriteBoolCell(bool boolean, std::streamsize cellWidthInChars);
 
 std::string_view ConvertMissionTypeToString(MissionTypes missionType);
 
-constexpr std::array<std::string_view, 7> columnTitles{
-	"DLL NAME",
-	"TYP",
-	"#",
-	"U",
-	"MAP NAME",
-	"TECH TREE NAME",
-	"MISSION DESCRIPTION",
+struct LegendEntry
```

Comment:
I initially implemented this as a std::pair instead of a custom struct. Ultimately, I felt it harmed readability and was better to go with the custom structure.